src/main/java/core/basesyntax/service/CsvFileReaderService.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax.service;
+
+import java.util.List;
+
+public interface CsvFileReaderService {
```

Comment:
```suggestion
public interface FileReaderService {
```
to make this interface more abstract